CMakeLists.txt

```diff
@@ -26,6 +26,7 @@ option(BUILD_TRANSLATIONS "" OFF)
 option(BUILD_SHARED_LIBS "" OFF)
 option(CHATTERINO_LTO "Enable LTO for all targets" OFF)
 option(CHATTERINO_PLUGINS "Enable EXPERIMENTAL plugin support in Chatterino" OFF)
+option(CHATTERINO_TEST_LOCAL_HTTPBIN "Use local httpbin docker image" ON)
```

Comment:
Define this option in the `tests/CMAKELISTS.txt` file instead (as far as I remember this should be possible)

Also inverse the option so it's something like `CHATTERINO_TEST_USE_PUBLIC_HTTPBIN` where the default is OFF